app/os/main.cpp

```diff
@@ -248,9 +252,35 @@ namespace display::internal {
 
 }	// namespace display::internal
 
-auto videokit	   = VideoKit {display::internal::event_flags, display::internal::corevideo};
+auto videokit = VideoKit {display::internal::event_flags, display::internal::corevideo};
```

Comment:
I don't understand why this is a relevant part of this PR but ok

Reply:
this is due to auto formatting because of the namespace at line 257